src/libraries/Common/src/System/Collections/Generic/EnumerableHelpers.Linq.cs

```diff
@@ -80,7 +80,9 @@ internal static T[] ToArray<T>(IEnumerable<T> source)
                 return result;
             }
 
-            var builder = new LargeArrayBuilder<T>(initialize: true);
+#pragma warning disable SA1129 // Do not use default value type constructor
+            LargeArrayBuilder<T> builder = new();
+#pragma warning restore SA1129 // Do not use default value type constructor
```

Comment:
```suggestion
            LargeArrayBuilder<T> builder = new();
```

Reply:
Ah, can't commit as a batch, will do it manually to prevent excess CI-triggering.